app/javascript/controllers/document_list_controller.js

```diff
@@ -0,0 +1,36 @@
+import { Controller } from 'stimulus';
+
+export default class extends Controller {
+  static targets = [ 'links', 'templateExternal', 'templateUploaded']
+
+  connect() {
+    this.wrapperClass = this.data.get('wrapperClass') || 'nested-fields';
+  }
+
+  addRecord(event) {
```

Comment:
This code here looks like it shouldn't be directly related with "documents", rather, it is more like generic add/remove resource logic, right?

Reply:
You are right. I will change that